client/src/components/PrioritizationCopy2/PrioritizationCopy2.tsx

```diff
@@ -127,12 +127,30 @@ const PrioritizationCopy2 =
      ) {
        // if 2-1
        if (percentTractTribal !== null && percentTractTribal == 0) {
-         prioCopy2Rendered = EXPLORE_COPY.getPrioFRTCopy(`less than 1%`, true);
+         prioCopy2Rendered = EXPLORE_COPY.getPrioFRTCopy(`less than 1%`, false);
          // if 2-2
        } else if (percentTractTribal !== null && percentTractTribal >= 0) {
-         prioCopy2Rendered = EXPLORE_COPY.getPrioFRTCopy(`${percentTractTribal}%`, true);
+         prioCopy2Rendered = EXPLORE_COPY.getPrioFRTCopy(`${percentTractTribal}%`, false);
        }
-       // if 3
+       // if 3-1
+     } else if (
```

Comment:
I don't understand why you need both of these two new elifs, since >=0 is also ==0, and tehy both call the same copy? Maybe I'm misreading one of the variables tho --- is it to handle percentTractTribal sometimes being a number? If so, would it maybe be simpler to just have the `>=` version and explicitly cast with `Number(percentTractTribal)` since that'll work if percentTractTribal is a number or a string?

Reply:
Nice catch @mattbowen-usds !! That should be >= 1!!

Fixing now!